ARCHITECTURE.md

```diff
@@ -133,6 +133,37 @@ The user is required to provide a closure to define the training process of the
 
 ## Inference
 
+```mermaid
+sequenceDiagram
+    participant Source
+    box rgb(255, 144, 0) CML
+    participant CML Core
+    participant Dataset
+    end 
+    Source --) CML Core: Arbitrary method of data transmission
+    Dataset ->> CML Core : Get the model information of the batch
+    CML Core ->> CML Core : Inference data
+    loop Each inference result
+        CML Core ->> Dataset: Add 1 ns as new timestamp
+    end
+    Dataset--)Database: Exec insertion statements
+```
+
+### Timestamp
+
+This is the same as mentioned above (see the section [above](#timestamp)).
```

Comment:
This section is unnecessary. Instead, maybe I'll move the previous one to the end of this document.